subprojects/sourcepawn/MessageNatives.cpp

```diff
@@ -19,6 +19,9 @@
 
 #include "ExtMain.hpp"
 
+bool ignoreHooks;
+TypeHandler<SPMod::IMessageHook> gMessageHooks;
```

Comment:
Ideally there should be only one global variable in the SourcePawn provider where we will store other global things like these.

+@Amaroq7 to discuss this.

Reply:
This could be done, but I'd leave it for another refactoring.